### DISCUSSION TITLE
A guide for new devs

### DISCUSSION THREAD

**Discussion by togfoxy** [togfoxy/MarsLander#202] [Ideas]
I'd like to write a small guide on the dev wiki that lets new devs understand the framework, file structure and some simple examples. Milon pointed out he feels the new code structured is beyond his ability.

Let's reduce the learning curve for me devs - both noobs and talented - by providing a kind of quick start guide. Things it can include:
- how to add a new image
- where new functions should go
- how main.lua calls other functions
- link to style guide
- what to expect when you do a PR (I.e a friendly code review)

Let's make the guide friendly and encouraging and highlight we are all learning and on a journey and elitism is not a requirement. :)

**Comment by MadByteDE** [comment]
I think I don't want to put time into this myself rn. Reviewing PRs, bug fixing and working on improvements is taking a lot of time for the last day or two.

Also I'd like to begin with implementing the lander-physics-imrpovements and already think about some stuff how to do it, but I'll like to wait for the main repo to be more stable.
